### DISCUSSION TITLE
Include path of a cmake rule

### DISCUSSION THREAD

**Discussion by LiyouZhou** [mvukov/rules_ros2#55] [Q&A] [answered]
@mvukov how did you determine what to put [here](https://github.com/mvukov/rules_ros2/blob/main/repositories/iceoryx.BUILD.bazel#L32). This path is not found in the source, nor in a normal cmake build. Does bazel build cmake targets in a weird folder structure?

**Comment by mvukov** [comment, marked as answer]
Hi Liyou, git blame says you introduced it in https://github.com/mvukov/rules_ros2/commit/8adc28ea55155be3e25509f03e5db4c98142c6ad. :) I think one needs to dive a bit under `bazel-bin/external/iceoryx/include` to figure out those things. Lemme know whether this helps.

**Reply by LiyouZhou** [reply]
You are right, it is strange where bazel decides to put things. or maybe it is to do with `make install`. But yes the source of truth is `bazel-bin/external/iceoryx/include`. thank you very much.
